src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_base_layout.html.twig

```diff
@@ -40,6 +40,16 @@
         <div {{ block('widget_container_attributes') }}>
             {{- form_errors(form.date) -}}
             {{- form_errors(form.time) -}}
+
+            <div style="display:none">
```

Comment:
in a Bootstrap layout, it would be better to use the bootstrap class to hide these rather than an inline style. CSP may forbid using inline styles.

Reply:
You are correct. https://getbootstrap.com/docs/4.0/getting-started/accessibility/#visually-hidden-content

Reply:
and btw, `display:none` hides it from screen readers too